flake.nix

```diff
@@ -0,0 +1,55 @@
+{
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.05";
+    utils.url = "github:gytis-ivaskevicius/flake-utils-plus";
+  };
+
+  outputs = inputs@{ self, utils, ... }:
+    utils.lib.mkFlake rec {
+      inherit self inputs;
+
+      supportedSystems = [
+        "aarch64-linux"
+        "aarch64-darwin"
+        "i686-linux"
+        "x86_64-darwin"
+        "x86_64-linux"
+      ];
+
+      outputsBuilder = channels: with channels; {
+        packages = with nixpkgs; { 
+          inherit (nixpkgs) package-from-overlays;
+
+          nixpacks = rustPlatform.buildRustPackage rec {
+            pname = "nixpacks";
+            version = "v0.0.20";
+            doCheck = true;
+            src = ./.;
+            nativeBuildInputs = [ clippy ];
+            # skip `cargo test` due tests FHS dependency
+            checkPhase = ''
+              cargo check
+              cargo clippy
+            '';
+            cargoLock = {
+              lockFile = ./Cargo.lock;
+            };
+            meta = with nixpkgs.lib; {
+              description = "App source + Nix packages + Docker = Image Resources";
```

Comment:
I think it's just "Image", not "Image Resources".

cc @somebody-who-knows-about-this

Reply:
```suggestion
              description = "App source + Nix packages + Docker = Image";
```

It is just image.